include/envoy/ratelimit/ratelimit.h

```diff
@@ -6,39 +6,14 @@
 #include <vector>
 
 #include "envoy/common/pure.h"
+#include "envoy/config/bootstrap/v2/bootstrap.pb.h"
```

Comment:
Why is bootstrap needed here? Anything in here should be an implementation detail I think.

Reply:
Ok. I think I can move the class `RateLimitServiceConfig` that I added to `configuration_impl.h`.